lib/query/index.js

```diff
@@ -61,11 +61,16 @@ function shouldRenderPrecalculatedData ({ statistics, count, extent }, { returnC
   return false
 }
 
-function renderPrecalculatedData (data, { returnCountOnly, returnExtentOnly }) {
+function renderPrecalculatedData (data, {
+  returnCountOnly,
+  returnExtentOnly,
+  outStatistics,
+  groupByFieldsForStatistics
+}) {
   const { statistics, count, extent } = data
 
   if (statistics) {
-    return renderPrecalculatedStatisticsResponse(data)
+    return renderPrecalculatedStatisticsResponse(data, { outStatistics, groupByFieldsForStatistics })
```

Comment:
Unless I'm mistaken, does the signature for `renderPrecalculatedStatisticsResponse()` have to change also? Cause looks like it's still taking in only one argument and that's `input`

Reply:
You're right, but it'll be in the followup PR.